### ISSUE TITLE
Support for multiple audio tracks

### ISSUE THREAD

**Issue by d9h02f** [llelectronics/videoPlayer#14]
Videos with more than one audio track (different languages, commentary, etc) currently only play back the first audio track, without any indication there are other audio tracks. Being able to select the audio track would be great.

**Comment by llelectronics** [comment]
Yes this would be wonderful. Sadly the gstreamer backend. API does not offer any way of accessing the different audio streams. So setting this as wontfix for now but keep it open. Maybe something changes in the future.
